app/models/invoice.rb

```diff
@@ -47,6 +47,25 @@ class Invoice < ActiveRecord::Base
 
   private
 
+  def self.period(period)
+    case period
+    when 'current-month'
+      where('invoices.ends_at >= ? AND invoices.ends_at <= ?', Time.now.beginning_of_month, Time.now)
```

Comment:
@yunixon why not use `invoices.ends_at between ? AND ?'` for all cases? I think it's more compact and readable.